src/hooks/users.ts

```diff
@@ -91,5 +126,7 @@ export const usePartygoers = (): readonly WithId<User>[] => {
 export const useUsersById = () => {
   useConnectUniverseUsers();
 
-  return useSelector(usersByIdSelector);
+  const universeUsersById = useSelector(usersByIdSelector);
+
+  return useMemo(() => universeUsersById, [universeUsersById]);
```

Comment:
We shouldn't need the `useMemo` here, as `useSelector` is always going to return the same data if it hasn't changed. So in effect, here we are currently memoing memoed data.